recipes/r-changepoints/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "r-changepoints" %}
+{% set version = "1.1.0" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/changepoints_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/changepoints/changepoints_{{ version }}.tar.gz
+  sha256: 73b7e389d8d5590d91ecb369f57b80f605b3c9f231da04be54a650b9b9c593f1
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - r-rcpp
+    - r-rcpparmadillo
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - libblas           # [not win]
+    - r-base
+    - r-gglasso
+    - r-glmnet
+    - r-ks
+    - r-mass
+    - r-data.tree
+    - r-rcpparmadillo
+    - liblapack        # [not win]
+  run:
+    - r-base
+    - r-gglasso
+    - r-glmnet
+    - r-ks
+    - r-mass
+    - r-data.tree
+    - r-rcpparmadillo
+    - {{ native }}gcc-libs         # [win]
+    - liblapack        # [not win]
+    - libstdcxx-ng       # [linux]
```

Comment:
Could you explain why the `libstdcxx-ng` and `libcxx` dependencies are needed at runtime? I've never seen this before.

Reply:
This is an indicator that you are missing a `compiler('cxx')` build dependency.